src/libraries/System.Private.CoreLib/src/System/AggregateException.cs

```diff
@@ -341,7 +341,7 @@ public override string Message
                     return base.Message;
                 }
 
-                StringBuilder sb = StringBuilderCache.Acquire();
+                var sb = new ValueStringBuilder(stackalloc char[256]);
```

Comment:
Honestly for this I wouldn't use `StringBuilderCache` or `ValueStringBuilder`. We're already in an exceptional code path; just use a regular uncached `StringBuilder` and let the GC handle it. Could also be slightly defensive against a corrupted cache being the thing that triggered the exception in the first place.